src/main/java/com/cleo/rest/services/impl/CarsWebService.java

```diff
@@ -1,30 +1,333 @@
 package com.cleo.rest.services.impl;
 
+import com.cleo.rest.exceptions.InvalidIdException;
 import com.cleo.rest.pojo.Car;
 import com.cleo.rest.services.ICarsWebService;
+import com.google.common.base.Strings;
 import com.google.common.collect.Lists;
+
+import java.net.URI;
+import java.util.Comparator;
 import java.util.List;
 import java.util.UUID;
+import java.util.stream.Collectors;
 import javax.ws.rs.core.Response;
 
+
+
+/**
+ * REST API exercise to perform CRUD operations on a list of cars.
+ *
+ *
+ * @author agesmer
+ * @since 7/10/2017
+ */
 public class CarsWebService implements ICarsWebService {
 
   // Don't worry about persisting anything, just do CRUD operations on this list.
   private final static List<Car> cars = Lists.newArrayList(
       new Car.Builder()
           .id(UUID.randomUUID())
+          .make("Toyota")
+          .model("Camry")
+          .color("Green")
+          .year(2010)
+          .build(),
+      new Car.Builder()
+          .id(UUID.randomUUID())
+          .make("Honda")
+          .model("CRV")
+          .color("Purple")
+          .year(2007)
+          .build(),
+      new Car.Builder()
+          .id(UUID.randomUUID())
+          .make("Buick")
+          .model("Encore")
+          .color("Black")
+          .year(2017)
+          .build(),
+      new Car.Builder()
+          .id(UUID.randomUUID())
+          .make("Buick")
+          .model("Enclave")
+          .color("Blue")
+          .year(2013)
           .build(),
       new Car.Builder()
           .id(UUID.randomUUID())
+          .make("Toyota")
+          .model("Prius")
+          .color("Red")
+          .year(2011)
           .build()
   );
-  
+
+  // How to handle bad input for filter & sort? Or even throw exception, since it will default?
+  /**
+   * Gets all cars. Default display is in ascending order of when cars were added to list. Optional query params are
+   * filter and sort.
+   *
+   *
+   * @param filter -> List cars by a specific attribute (id, make, model, color, year).
+   * @param sort -> Specify order of cars based on filter to be in ascending (asc) or descending (desc) order.
+   * @return -> Display of cars based on user's query parameters.
+   */
```

Comment:
This Javadoc is pretty good. Don't worry about changing it here, but in production code this would generally go on the interface.